<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fviewMainWindow</class>
 <widget class="QMainWindow" name="fviewMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>808</width>
    <height>611</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CMTK Fusion Viewer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,1,0,0,0,0" columnstretch="1,1">
      <property name="sizeConstraint">
       <enum>QLayout::SetMaximumSize</enum>
      </property>
      <item row="4" column="0">
       <widget class="QSlider" name="blackSliderFix">
        <property name="maximum">
         <number>499</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QSlider" name="blackSliderMov">
        <property name="maximum">
         <number>499</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="bwLabelFix">
        <property name="text">
         <string>Fixed Image Black / White</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="bwLabelMov">
        <property name="text">
         <string>Moving Image Black / White</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QGroupBox" name="fixedImgGroupBox">
        <property name="title">
         <string>Fixed Image</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="1">
          <widget class="QGraphicsView" name="fixedView">
           <property name="cursor" stdset="0">
            <cursorShape>CrossCursor</cursorShape>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QGroupBox" name="movingImgGroupBox">
        <property name="title">
         <string>Moving/Fused Image</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="0">
          <widget class="QGraphicsView" name="movingView">
           <property name="cursor" stdset="0">
            <cursorShape>CrossCursor</cursorShape>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QSlider" name="whiteSliderMov">
        <property name="maximum">
         <number>499</number>
        </property>
        <property name="value">
         <number>499</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="alphaLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Transparency</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="labelAlphaFixed">
          <property name="text">
           <string>Fixed Image</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="alphaSlider">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelAlphaMoving">
          <property name="text">
           <string>Moving Image</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QSlider" name="sliceSlider">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,1,0">
        <item>
         <widget class="QLabel" name="sliceLabelFrom">
          <property name="text">
           <string>I</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="sliceLabel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Slice</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="sliceLabelTo">
          <property name="text">
           <string>S</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="5" column="0">
       <widget class="QSlider" name="whiteSliderFix">
        <property name="maximum">
         <number>499</number>
        </property>
        <property name="value">
         <number>499</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="enabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>808</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_Quit"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <widget class="QMenu" name="menuSlice_Orientation">
     <property name="title">
      <string>Slice &amp;Orientation</string>
     </property>
     <addaction name="actionSliceAxial_XY"/>
     <addaction name="actionSliceCoronal_XZ"/>
     <addaction name="actionSliceSagittal_YZ"/>
    </widget>
    <widget class="QMenu" name="menu_Zoom">
     <property name="title">
      <string>&amp;Zoom</string>
     </property>
     <addaction name="actionZoom25"/>
     <addaction name="actionZoom50"/>
     <addaction name="separator"/>
     <addaction name="actionZoom100"/>
     <addaction name="separator"/>
     <addaction name="actionZoom200"/>
     <addaction name="actionZoom300"/>
     <addaction name="actionZoom400"/>
    </widget>
    <widget class="QMenu" name="menuInterpolation">
     <property name="title">
      <string>Interpolation</string>
     </property>
     <addaction name="actionInterpLinear"/>
     <addaction name="actionInterpCubic"/>
     <addaction name="actionInterpSinc"/>
     <addaction name="separator"/>
     <addaction name="actionInterpNearestNeighbour"/>
     <addaction name="actionInterpPartialVolume"/>
    </widget>
    <widget class="QMenu" name="menuAnnotations">
     <property name="title">
      <string>&amp;Annotations</string>
     </property>
     <addaction name="actionLinkedCursor"/>
    </widget>
    <addaction name="menu_Zoom"/>
    <addaction name="separator"/>
    <addaction name="menuInterpolation"/>
    <addaction name="menuSlice_Orientation"/>
    <addaction name="separator"/>
    <addaction name="menuAnnotations"/>
   </widget>
   <widget class="QMenu" name="menuTransform">
    <property name="title">
     <string>&amp;Transform</string>
    </property>
    <addaction name="actionAffineOnly"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menuView"/>
   <addaction name="menuTransform"/>
  </widget>
  <action name="action_Quit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
  </action>
  <action name="action_Zoom">
   <property name="text">
    <string>&amp;Zoom</string>
   </property>
  </action>
  <action name="actionSliceAxial_XY">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Axial (XY)</string>
   </property>
   <property name="shortcut">
    <string>Alt+Z</string>
   </property>
  </action>
  <action name="actionSliceCoronal_XZ">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Coronal (XZ)</string>
   </property>
   <property name="shortcut">
    <string>Alt+Y</string>
   </property>
  </action>
  <action name="actionSliceSagittal_YZ">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Sagittal (YZ)</string>
   </property>
   <property name="shortcut">
    <string>Alt+X</string>
   </property>
  </action>
  <action name="actionZoom100">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;100%</string>
   </property>
   <property name="shortcut">
    <string>Alt+1</string>
   </property>
  </action>
  <action name="actionZoom200">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;200%</string>
   </property>
   <property name="shortcut">
    <string>Alt+2</string>
   </property>
  </action>
  <action name="actionZoom400">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;400%</string>
   </property>
   <property name="shortcut">
    <string>Alt+4</string>
   </property>
  </action>
  <action name="actionZoom50">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;50%</string>
   </property>
   <property name="shortcut">
    <string>Alt+5</string>
   </property>
  </action>
  <action name="actionZoom300">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;300%</string>
   </property>
   <property name="shortcut">
    <string>Alt+3</string>
   </property>
  </action>
  <action name="actionZoom25">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>25%</string>
   </property>
  </action>
  <action name="actionInterpLinear">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Linear</string>
   </property>
   <property name="shortcut">
    <string>Alt+L</string>
   </property>
  </action>
  <action name="actionInterpCubic">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Cubic</string>
   </property>
   <property name="shortcut">
    <string>Alt+C</string>
   </property>
  </action>
  <action name="actionInterpSinc">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Sinc</string>
   </property>
   <property name="shortcut">
    <string>Alt+S</string>
   </property>
  </action>
  <action name="actionInterpNearestNeighbour">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Nearest Neighbour</string>
   </property>
   <property name="shortcut">
    <string>Alt+N</string>
   </property>
  </action>
  <action name="actionInterpPartialVolume">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Partial Volume</string>
   </property>
   <property name="shortcut">
    <string>Alt+P</string>
   </property>
  </action>
  <action name="actionLinkedCursor">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Linked &amp;Cursor</string>
   </property>
  </action>
  <action name="actionAffineOnly">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Affine Only</string>
   </property>
   <property name="shortcut">
    <string>Alt+A</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>action_Quit</sender>
   <signal>activated()</signal>
   <receiver>fviewMainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>403</x>
     <y>305</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
